compiler/z/codegen/S390GenerateInstructions.hpp

```diff
@@ -1511,15 +1511,79 @@ TR::Instruction *
 generateReplicateNodeInVectorReg(TR::Node * node, TR::CodeGenerator *cg, TR::Register * targetVRF, TR::Node * srcElementNode,
                                  int elementSize, TR::Register *zeroReg=NULL, TR::Instruction * preced=NULL);
 
+/**
+ * \param node
+ *    The node being processed.
+ *
+ * \param cg
+ *    The code generator used to generate the instructions.
+ * 
+ * \param targetRegister
+ *    The register where the specific shifted bits from sourceRegister will be placed to the corresponding 
+ *    location. The remaining bits will be zeroed out.
+ * 
+ * \param sourceRegister
+ *    The register which will be shifted. The selected shifted bits in sourceRegister will be placed to 
+ *    targetRegister, and the sourceRegister itself will stay unchanged. 
+ * 
+ * \param fromBit
+ *    indicates the starting bit position of the selected range of bits in targetRegister and in sourceRegister after shifting. 
+ * 
+ * \param toBit
+ *    indicates the ending bit position of the selected range of bits in targetRegister and in sourceRegister after shifting. 
+ * 
+ * \param shiftAmount
+ *    indicates the amount of bits that sourceRegister is shifted to the left. 
+ * 
+ * \param setConditionCode
+ *    true if we are generating the RISBG instruction; false otherwise.  
```

Comment:
I don't see anywhere in the codebase where we pass `true` to this argument. Can we remove it all together?